compel/include/uapi/infect.h

```diff
@@ -120,6 +120,7 @@ struct infect_ctx {
 	open_proc_fn open_proc;
 
 	int log_fd; /* fd for parasite code to send messages to */
+	unsigned long user_set_remote_map;
```

Comment:
nit: I think it is better to rename it to remote_map_addr and add a comment to explain how it can be used.

Reply:
fixed